tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-cad-module/src/main/java/org/apache/tika/parser/dwg/DWGParser.java

```diff
@@ -128,7 +135,7 @@ public void parse(InputStream stream, ContentHandler handler, Metadata metadata,
                 throw new TikaException("Unsupported AutoCAD drawing version: " + version);
         }
 
-        xhtml.endDocument();
+        xhtml.endDocument();}
```

Comment:
formatting.  Did checkstyle let this one through?

Reply:
should now be fixed